src/components/OpenSource/Partnerships.js

```diff
@@ -9,7 +10,15 @@ import Image from '../Common/Image'
 import SubtitleWithBody from '../Common/SubtitleWithBody'
 
 const StyledSectionTitle = styled(Col)`
```

Comment:
Why do you need all of these paddings on top of the `Col` component's `width` prop? The spacing should be handled by the grid :confused:

Reply:
Done with the Grid. Thanks Jen